.eslintrc.js

```diff
@@ -0,0 +1,228 @@
+module.exports = {
+  "env": {
+    "browser": true,
+    "jest/globals": true
+  },
+  "extends": [
+    "plugin:@typescript-eslint/recommended",
+    "plugin:@typescript-eslint/recommended-requiring-type-checking",
+    "prettier",
+    "plugin:jest/recommended"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "tsconfigRootDir": __dirname,
+    "project": ["./tsconfig.json", ],
+    "sourceType": "module"
+  },
+  "ignorePatterns": ["**/__tests__/*"],
+  "plugins": [
+    "eslint-plugin-jsdoc",
+    "eslint-plugin-prefer-arrow",
+    "eslint-plugin-react",
+    "@typescript-eslint",
+    "jest"
+  ],
+  "rules": {
```

Comment:
Where do all these rules come from?

Reply:
`eslint --init` :D removed most of them, I have yet to figure out some issues with estlint-config-jitsi